AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/AWSIncomingEventReconciliationQueue.swift

```diff
@@ -16,17 +16,19 @@ typealias IncomingEventReconciliationQueueFactory =
     ([ModelSchema],
     APICategoryGraphQLBehavior,
     StorageEngineAdapter,
+    DataStoreConfiguration,
     AuthCategoryBehavior?,
     ModelReconciliationQueueFactory?
 ) -> IncomingEventReconciliationQueue
 
 @available(iOS 13.0, *)
 final class AWSIncomingEventReconciliationQueue: IncomingEventReconciliationQueue {
 
-    static let factory: IncomingEventReconciliationQueueFactory = { modelSchemas, api, storageAdapter, auth, _ in
+    static let factory: IncomingEventReconciliationQueueFactory = { modelSchemas, api, storageAdapter, config, auth, _ in
```

Comment:
The queue shouldn't have to know about all of the configuration, only about those elements related to its operation

Reply:
Ah, good call, i will update!

Reply:
Oops my bad, didn't realize we could send in an array of `DataStoreSyncExpression` 's!  Thanks for catching this!!  updated in the latest revision.